iosApp/iosApp/ui/pages/RegistrationScreen.swift

```diff
@@ -144,3 +147,35 @@ struct CarConnectStep: View {
     }
 }
 
+struct PermissionStep: View {
+    let strings = StringsHelper().getResourceStrings()
+    var onAction: (RegisterScreenUiAction) -> Void
+    let center = UNUserNotificationCenter.current()
+    
+    var body: some View {
+        VStack(alignment: .center) {
+            Text(strings.get(id: SharedRes.strings().allow_notis, args: []))
+                .padding(.top, 89)
+                .fontWeight(.bold)
+                .font(.system(size: 28))
+                .multilineTextAlignment(.center)
+            
+            Text(strings.get(id: SharedRes.strings().noti_info, args: []))
+                .padding(.bottom, 43)
+                .multilineTextAlignment(.center)
+            
+            PrimaryButton(text: strings.get(id: SharedRes.strings().allow_notis, args: []), enabled: .constant(true), action: {
```

Comment:
Just wondering, do you always have to put args: []? Couldn't this be a default value? The same for the enabled: true. Could be that this is how it's written in Swift but it just seems a bit odd.

Reply:
I read that it's possible to give default arguments, however, when I do not add the args: [] it starts complaining and giving me errors saying it has to be provided. Maybe this is because its calling a kotlin function in swift?